crates/bevy_pbr/src/render/mesh_bindings.rs

```diff
@@ -34,9 +37,12 @@ mod layout_entry {
             },
         }
     }
-    pub(super) fn model(binding: u32) -> BindGroupLayoutEntry {
-        let size = MeshUniform::min_size().get();
-        buffer(binding, size, ShaderStages::VERTEX | ShaderStages::FRAGMENT)
+    pub(super) fn model(render_device: &RenderDevice, binding: u32) -> BindGroupLayoutEntry {
+        GpuArrayBuffer::<MeshUniform>::binding_layout(
+            binding,
+            ShaderStages::VERTEX_FRAGMENT,
+            render_device,
```

Comment:
In a future PR, it might be worthwhile to newtype `device.limits().max_storage_buffers_per_shader_stage`, and accept that newtype instead of `RenderDevice` as argument to `GpuArrayBuffer` so that it's clear what data it is using.

Reply:
Mmm. I thought about passing the values explicitly, but then I also thought that maybe other features or limits may impact logic at some point. If it had been more difficult to plumb it in then I would have changed it.